webapp/graphite/settings.py

```diff
@@ -95,6 +95,11 @@
 LOG_ROTATION = True
 LOG_ROTATION_COUNT = 1
 
+LOG_FILE_INFO = 'info.log'
```

Comment:
Maybe add a link to https://docs.djangoproject.com/en/1.11/topics/logging/#examples ?

Reply:
Does it applicable here?

Reply:
Yes, it's a logging/django thing, it always work :)

Reply:
It is linked on line 92